common/src/main/java/com/revenuecat/purchases/common/ReplaceProductInfo.kt

```diff
@@ -5,5 +5,5 @@ import com.revenuecat.purchases.models.StoreTransaction
 
 data class ReplaceProductInfo(
     val oldPurchase: StoreTransaction,
-    @BillingFlowParams.ProrationMode val prorationMode: Int
+    @BillingFlowParams.ProrationMode val prorationMode: Int? = null
```

Comment:
Does this need to be nullable now? 🤔

Reply:
yeah, to avoid breaking changes from the previous version :/ i'll triply check against main though to confirm..

Reply:
yeah, the deprecated methods allow for no prorationmode to be passed in, and we defaulted to null. 

wondering if it's worth creating a whole ticket to remember everything to update when we are able to remove the old purchase methods..